packages/ui/src/atom/label-tooltip-atom/label-tooltip-atom.theme-default.scss

```diff
@@ -0,0 +1,15 @@
+.label-tooltip-atom.theme-default {
+  display: inline-block;
+  vertical-align: middle;
+}
+.label-tooltip-atom.theme-default .label-tooltip-atom-icon {
+  display: inline-block;
+  font-size: 12px;
+  line-height: 14px;
```

Comment:
Use `line-height` ONLY in relative units. The best case is multiplier.
For example, if you see `line-height: 14px` and `font-size: 12px` in design tool, you should add `line-height: 1.16`.
If you would define `line-height` with any unit - font resize will work incorrectly.